projects/plugins/jetpack/modules/woocommerce-analytics/classes/class-jetpack-woocommerce-analytics-checkout-flow.php

```diff
@@ -147,18 +147,27 @@ public function capture_checkout_view() {
 		$is_checkout = $checkout_page_id && is_page( $checkout_page_id )
 		|| wc_post_content_has_shortcode( 'woocommerce_checkout' )
 		|| has_block( 'woocommerce/checkout', $post )
+		|| has_block( 'woocommerce/classic-shortcode', $post )
```

Comment:
There is no way to check if the block is Cart or Checkout.

Reply:
True, this is OK though. It's based on the template as far as I can tel. If someone has set up their Checkout page to use the Cart template then it will be an edge case, don't think we can or should try to solve for this.